From 30bf2af0ec14c1b5b7c0db71095da1c0a397d2d6 Mon Sep 17 00:00:00 2001
From: Gabriele Vedovato <Gabriele.Vedovato@lnl.infn.it>
Date: Sat, 30 Aug 2025 08:03:36 -0700
Subject: [PATCH 3/6] add '#pragma link C++ class xTDocDirective+;'

---
 tools/xhtml/xhtml_LinkDef.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/xhtml/xhtml_LinkDef.h b/tools/xhtml/xhtml_LinkDef.h
index 699bff11..9dec747a 100644
--- a/tools/xhtml/xhtml_LinkDef.h
+++ b/tools/xhtml/xhtml_LinkDef.h
@@ -22,5 +22,6 @@
 #pragma link C++ class xTDocParser;
 #pragma link C++ class xTDocInfo+;
 #pragma link C++ class xTClassDocInfo+;
+#pragma link C++ class xTDocDirective+;
 
 #endif // __CLING__
-- 
2.39.5

