From bed91c7b60879504f504ea5f007b9e9fa68f7366 Mon Sep 17 00:00:00 2001
From: Francesco Salemi <francesco.salemi@uniroma1.it>
Date: Sat, 30 Aug 2025 09:02:03 -0700
Subject: [PATCH 5/7] Minor changes

---
 tools/xhtml/xTDocDirective.cc | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/tools/xhtml/xTDocDirective.cc b/tools/xhtml/xTDocDirective.cc
index c67cdf67..342dffce 100644
--- a/tools/xhtml/xTDocDirective.cc
+++ b/tools/xhtml/xTDocDirective.cc
@@ -2,7 +2,7 @@
 
 #include "TApplication.h"
 #include "TClass.h"
-#include "TDocInfo.h"
+#include "xTDocInfo.hh"
 #include "xTDocOutput.hh"
 #include "xTDocParser.hh"
 #include "TError.h"
@@ -294,14 +294,14 @@ void xTDocMacroDirective::SubProcess(const TString& what, const TString& out) {
    if (!gROOT->IsBatch()) {
       // to get X11 to sync :-( gVirtualX->Update()/Sync() don't do it
       gSystem->Sleep(1000);
-      gVirtualX->Update(0);
-      gVirtualX->Update(1);
+      //gVirtualX->Update(0);
+      //gVirtualX->Update(1);
    }
 
    gSystem->ProcessEvents();
    if (!gROOT->IsBatch()) {
-      gVirtualX->Update(0);
-      gVirtualX->Update(1);
+      //gVirtualX->Update(0);
+      //gVirtualX->Update(1);
    }
 
    objRet->SaveAs(out);
@@ -505,7 +505,7 @@ Bool_t xTDocMacroDirective::GetResult(TString& result)
 
       xTDocParser *dparser = 0;
       if (GetDocParser()->GetCurrentClass())
-         dparser = new xTDocParser(*(TClassDocOutput*)GetDocOutput(), GetDocParser()->GetCurrentClass());
+         dparser = new xTDocParser(*(xTClassDocOutput*)GetDocOutput(), GetDocParser()->GetCurrentClass());
       else dparser = new xTDocParser(*GetDocOutput());
       std::stringstream ssConverted;
       dparser->Convert(ssConverted, ssRaw, "./", kTRUE /*code*/, kFALSE /*process directives*/);
-- 
2.39.5

